fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java

```diff
@@ -158,19 +147,19 @@
     private Result<JobUpdateResults, OToolError> bumpPlatform(Map<String, List<String>> paramsMap) {
         final Optional<String> fromOptional = extractParamValue(paramsMap, "from");
         final Optional<String> toOptional = extractParamValue(paramsMap, "to");
-        final Optional<List<String>> projectsOptional = extractParamValues(paramsMap, "projects");
```

Comment:
why change this? Javalin splits the values for you

Reply:
no it does not :))) You completly misunderstood that usage